accepted/2020/windows-specific-apis/windows-specific-apis.md

```diff
@@ -397,3 +838,62 @@ Windows-specific:
     - `TryOpenExisting(String, Semaphore)`
 * **Thread**
     - `SetApartmentState(ApartmentState)`
+
+### System.Xaml.Permissions
+
+* **XamlAccessLevel**
+    - `AssemblyAccessTo(Assembly)`
+    - `AssemblyAccessTo(AssemblyName)`
+    - `get_AssemblyAccessToAssemblyName()`
+    - `get_PrivateAccessToTypeName()`
+    - `PrivateAccessTo(String)`
+    - `PrivateAccessTo(Type)`
+
+### Microsoft.VisualBasic
```

Comment:
@jaredpar do these annotations look right to you?

Reply:
cc @cston

Reply:
@jaredpar signed off, so I assume this is good. @cston, let me know if you have any concerns.